e2e-tests/tfm/cypress/e2e/journeys/case-amendments/show-amendment-button/deal-cancellation-enabled/show-amendment-button.spec.js

```diff
@@ -0,0 +1,38 @@
+import relative from '../../../../relativeURL';
+import facilityPage from '../../../../pages/facilityPage';
+import amendmentsPage from '../../../../pages/amendments/amendmentsPage';
+import MOCK_DEAL_AIN from '../../../../../fixtures/deal-AIN';
+import { T1_USER_1, PIM_USER_1, BANK1_MAKER1 } from '../../../../../../../e2e-fixtures';
+import { submitDealCancellation } from '../../../../../support/trade-finance-manager-ui/submit-deal-cancellation';
+
+context('Amendments page', () => {
+  describe('A cancelled deal', () => {
```

Comment:
Can we rename to be more descriptive and no describe, e.g

```js
context('Amendments page - cancelled deal - X feature flag enabled', () => {
```

Reply:
And the file name I think shoudl just be `amendment-button.spec.js`

Reply:
I've kept the describe block in matchiong with the non ff test file -- but done the rest (and renamed the non ff test file)